pkg/distributor/distributor.go

```diff
@@ -98,8 +107,13 @@ type Distributor struct {
 // create a Distributor
 type Config struct {
 	EnableBilling bool
+	HAReplicas    bool
+	HARejectOne   bool
+	KVStore       string
```

Comment:
These should be part of `TrackerConfig`, no?

Reply:
I think it makes sense to keep `HAReplicas` as part of distributor config. The rest can move.